packages/components/src/styles/list/list.stories.ts

```diff
@@ -12,8 +12,8 @@ const { generateTemplate } = storybookTemplate('sd-list');
  * Text lists can be numbered, have bullet points, or be supplemented by content symbols. Text can be bolded or linked.
  *
  * **Related templates**:
- * - [List with Bolded Text](?path=/docs/templates-list-with-bolded-text--docs)
- * - [Link list](?path=/docs/templates-link-list--docs)
+ * - [List](?path=/docs/templates-list--docs)
+ * - [Link list](?path=/docs/templates-link--docs&globals=backgrounds.value:transparent#link-list)
```

Comment:
Please check my comment on th
```suggestion
 * - [Link list](?path=/docs/templates-link-list--docs#link-list)
```

Reply:
Updated